ArcGISRuntimeSDKQt_CppSamples/EditData/ContingentValues/ContingentValues.qml

```diff
@@ -0,0 +1,210 @@
+
+                onCurrentValueChanged: {
+                    if (protectionComboBox.currentValue === "")
+                        return;
+
+                    // Update the feature's attribute map with the selection
+                    contingentValuesSample.updateField("Protection", protectionComboBox.currentValue);
+
+                    // Get the valid contingent range values for the subsequent SpinBox
+                    const minMax = contingentValuesSample.getContingentValues("BufferSize", "BufferSizeFieldGroup")
+
+                    // If getContingentValues() returned results, update the spin box values
+                    if (minMax[0] !== "") {
+                        rangeValuesSpinBox.from = minMax[0];
+                        rangeValuesSpinBox.to = minMax[1];
+
+                        // If the maxValue in the range is 0, set the buffer size to 0
+                        if (minMax[1] === 0) {
+                            contingentValuesSample.updateField("BufferSize", 0);
+                            saveButton.enabled = contingentValuesSample.validateContingentValues();
+                        }
+                    }
+                }
+            }
+
+            Text {
+                text: "Buffer Size"
+                font {
+                    bold: true
+                    pointSize: 11
+                }
+            }
+
+            Text {
+                text: rangeValuesSpinBox.from + " to " + rangeValuesSpinBox.to;
+            }
+
+            SpinBox {
+                id: rangeValuesSpinBox
+                editable: true
+                from: 0;
+                to: 0;
+                stepSize: 10
+                value: 0;
+
+                enabled: protectionComboBox.currentText !== ""
+
+                onValueChanged: {
+                    if (protectionComboBox.currentValue === "")
+                        return;
+
+                    contingentValuesSample.updateField("BufferSize", rangeValuesSpinBox.value);
+
+                    // Validate that all contingencies are valid, if so, enable the save button
+                    saveButton.enabled = contingentValuesSample.validateContingentValues();
+                }
+            }
+
+            Button {
+                id: saveButton
+                Text {
+                    anchors.fill: parent
+                    horizontalAlignment: Text.AlignHCenter
+                    verticalAlignment: Text.AlignVCenter
+                    text: "Save"
+                }
+
+                enabled: false
+
+                onClicked: {
+                    const valid = contingentValuesSample.validateContingentValues();
+                    if (valid) {
+                        contingentValuesSample.createNewNest();
+                        contingentValuesSample.featureAttributesPaneVisibe = false;
+                    }
+                }
+            }
+
+            Button {
+                id: discardButton
```

Comment:
Is it possible to also remove this pane when user clicks elsewhere in the map instead of just relying on clicking discard?